frontend/src/main/scala/akkaviz/frontend/vis/package.scala

```diff
@@ -0,0 +1,286 @@
+    }
+  }
+
+  @js.native
+  trait NetworkOptions extends js.Any
+
+  object NetworkOptions {
+    //TODO: define typesafe builder
+    def apply(): NetworkOptions = {
+      js.Dynamic.literal().asInstanceOf[NetworkOptions]
+    }
+  }
+
+  @js.native
+  trait Coords extends js.Any {
+    val x: Double
+    val y: Double
+  }
+
+  object Coords {
+    def apply(x: Double, y: Double): Coords = {
+      js.Dynamic.literal(x = x, y = y).asInstanceOf[Coords]
+    }
+  }
+
+  @js.native
+  trait NetworkMoveToOptions extends js.Any {
+    var position: Coords = js.native
+    var scale: Double = js.native
+    var offset: Coords = js.native
+    //var animation
+  }
+
+  @js.native
+  trait NetworkFitOptions extends NetworkMoveToOptions {
+  }
+
+  @js.native
+  trait NetworkFocusOptions extends NetworkMoveToOptions {
+    var locked: Boolean = js.native
+  }
+
+  @js.native
+  trait SelectionOptions extends js.Any {
+    var unselectAll: Boolean = js.native
+    var highlightEdges: Boolean = js.native
+  }
+
+  object SelectionOptions {
+    def apply(unselectAll: js.UndefOr[Boolean] = js.undefined, highlightEdges: js.UndefOr[Boolean] = js.undefined) = {
+      js.Dynamic.literal(unselectAll = unselectAll, highlightEdges = highlightEdges).asInstanceOf[SelectionOptions]
+    }
+  }
+
+  @js.native
+  @JSName("vis.Network")
+  class Network(element: Element, data: NetworkData, options: NetworkOptions) extends js.Any {
+
+    //Global methods for the network
+    def destroy(): Unit = js.native
+
+    def setData(data: NetworkData): Unit = js.native
+
+    def setOptions(options: NetworkOptions): Unit = js.native
+
+    def on(event: String, fn: js.Function): Unit = js.native
+
+    def off(event: String, fn: js.Function): Unit = js.native
+
+    def once(event: String, fn: js.Function): Unit = js.native
+
+    //Methods related to the canvas
+    def canvasToDOM(coords: Coords): Coords = js.native
+
+    def DOMtoCanvas(coords: Coords): Coords = js.native
+
+    def redraw(): Unit = js.native
+
+    def setSize(width: String, height: String): Unit = js.native
+
+    //Methods to get information on nodes and edges.
+
+    //Physics methods to control when the simulation should run.
+
+    //Selection methods for nodes and edges.
+    def getSelection(): Selection = js.native
+
+    def getSelectedNodes(): js.Array[String] = js.native
+
+    def getSelectedEdges(): js.Array[String] = js.native
+
+    def getNodeAt(domPosition: Coords): String = js.native
+
+    def getEdgeAt(domPosition: Coords): String = js.native
+
+    def selectNodes(nodeIds: js.Array[String], highlightEdges: Boolean = ???): Unit = js.native
+
+    def selectEdges(edgeIds: js.Array[String]): Unit = js.native
+
+    def setSellection(selection: Selection, options: js.UndefOr[SelectionOptions] = js.undefined): Unit = js.native
+
+    def unselectAll(): Unit = js.native
+
+    //Methods to control the viewport for zoom and animation.
+    def getScale(): Double = js.native
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Do not prefix getters with 'get'](https://www.codacy.com/app/lustefaniak/akka-viz/file/2162074181/issues/source?bid=3121282&fileBranchId=3158877#l177)